### ISSUE TITLE
Registration Failed - Webconnect

### ISSUE THREAD

**Issue by RydalWater** [Loopring/dexwebapp#184]
Hi, I'm having an issue where it doesn't seem possible to register for an account on loopring.io with wallet connect. The device successfully connects no problem. I click on Register and it add in the CoinGecko "Loopring zkRollup VIP Starter Kit" code that I just purchased to the promo code box. Click OK and it just pops up with "Something Went Wrong, Please try to Register again".

I'm reluctant to step forward from here without the code as I have a feeling that I wont be able to add it later.

I've also received two of the following emails:

Run failed for master (503bfe0)
Repository: Loopring/dexwebapp
Workflow:
Duration: 1.0 second
Finished: 2020-08-06 10:31:19 UTC
https://github.com/Loopring/dexwebapp/actions/runs/197561446

**Separate issue/Comment:**
Additionally, I already have an account with a different wallet and would have used the code on this one. However, it seem that I opened that account a while ago and the login system has since changed the system. Unfortunately, I will have to pay an unreasonable gas fee to unlock the account (meaning that my funds which are left in the DEX are likely gone). Really not sure this was a suitable method of going about re-registration without reimbursing the gas fees.

**Comment by Mynima** [comment]
Still waiting on a response, let me know what is happening please. Did I waste spending on this VIP starter kit for Loopring?
